1_space_odyssey/js/luke-9000/app.js

```diff
@@ -0,0 +1,209 @@
+  /   _.  |    ;
+ ;-.-'|    \   |
+/   | \    _\  _\
+\__/'._;.  ==' ==\
+         \    \   |
+         /    /   /
+         /-._/-._/
+  ...    \   `\  \
+          `-._/._/
+
+
+ 1. CREATE DROID::::::
+    Below are all the variables needed to initialize an android into consciousness in its default state.
+    a. assign the correct values to the variables below,
+    b. ensure to choose the correct values and datatypes
+    c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
+*/
+var isAlive = true;
+var name = "Doctor John";
+var hasJetPack = true;
+var suitColor = "white and red";
+var eyes = 2;
+var eyeColor = "blue";
+var hairColor = "pink";
+var legs = 2;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = false;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log("\u{1F916} .-.-. Booting .-.-.");
+console.log(
+    name,
+    isAlive,
+    hasJetPack,
+    eyes,
+    eyeColor,
+    hairColor,
+    legs,
+    arms,
+    hands,
+    ears,
+    canWalk,
+    canFly,
+    hasWeapon
+);
+
+/*
+ 2. BROKEN SATELLITE::::::
+    You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
+    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid
+       cannot go outside, console log the correct log based on that condition.
+*/
+console.log(
+    "\u{1F6F0} BROKEN SATELLITE::::::"
+);
+
+/*
+ uncomment these console logs and write your if else statement and use them to console log out the correct statement
+*/
+
+if (hasJetPack === true) {
+    console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+} else {
+    console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
+}
+
+
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log(
+    "\u{1F327} METEOR SHOWER::::::"
+);
+
+var responses = [
+    "Call basecamp and ask for assistance, but they are far away",
+    "Do nothing, just push through",
+    "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+    "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+];
+
+
+// write your code here, use the console log beneath
+
+var nrOfResponses = responses.length;
+
+for (var i = 0; i <= nrOfResponses; i++) {
+    if (responses[i] === responses[2]) {
+        console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+    } else {console.log("Ship has been kidnapped by aliens")}
```

Comment:
clean code, format an if else should look like:

```js
if (condition) {
  ...
} else {
  ...
}
```

notice the whitespace and lines